packer/vagrant/scripts/windows/msys2-install.bat

```diff
@@ -2,7 +2,7 @@ if not exist "C:\Windows\Temp\msys2.exe" (
   powershell -Command "[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::TLS12; (New-Object System.Net.WebClient).DownloadFile('https://repo.msys2.org/distrib/x86_64/msys2-x86_64-20210228.exe', 'C:\Windows\Temp\msys2.exe')" <NUL
 )
 
-cmd /c "C:\Windows\Temp\msys2.exe -v --platform minimal --script A:\msys2-install.qs"
+cmd /c "C:\Windows\Temp\msys2.exe -d --platform minimal --script A:\msys2-install.qs"
```

Comment:
just curious what the `-v` to `-d` is here. output option?

Reply:
`-v` prints out the version, `-d` prints out the debug log